Gemfile

```diff
@@ -1,53 +1,25 @@
 source 'https://rubygems.org'
```

Comment:
@vad1m777  Why you touch this file?

Reply:
In tutorial write:
'Important note: For all the Gemfiles in this book, you should use the version numbers listed at gemfiles-4th-ed.railstutorial.org instead of the ones listed below (although they should be identical if you are reading this online).'

Reply:
@viktordanch 
Important note: For all the Gemfiles in this book, you should use the version numbers listed at gemfiles-4th-ed.railstutorial.org instead of the ones listed below (although they should be identical if you are reading this online).